src/averageAges.js

```diff
@@ -37,7 +50,19 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  let filteredWomen = people.filter((person) => {
+    return person.sex === 'f';
+  });
```

Comment:
```suggestion
  let filteredWomen = people.filter(person =>  person.sex === 'f');
```

Reply:
You don't need to write a condition in curly braces if you have just a simple one.

Reply:
Good tip, please fix